frontend/src/App.js

```diff
@@ -137,7 +137,7 @@ function App() {
           <Header />
           <SiteNav admin={admin} authenticated={authenticated} logout={logout} user={user} />
           <div className="grid-row maxw-widescreen flex-align-start smart-hub-offset-nav tablet:smart-hub-offset-nav desktop:smart-hub-offset-nav margin-top-9">
-            <div className="grid-col-12 margin-top-2 margin-right-2">
+            <div className="grid-col-12 margin-top-2 margin-right-2 margin-left-3">
```

Comment:
I added the same class to just the landing page (`src/pages/Landing/index.js` around line 328) so the landing page is probably pushed a little to far to the right.

Reply:
Sounds good.